core/modules/system/system.module

```diff
@@ -2595,6 +2595,12 @@ function system_user_timezone(&$form, &$form_state) {
 
 /**
  * Implements hook_block_info().
+ *
+ * Note that this also returns the system-defined menu blocks (as opposed to any
```

Comment:
I don't think we need this, as any module can define menu blocks. There's nothing special about System module. (I do like the new docblock for Menu module though, because people will probably look there first for menu blocks.